base/stats.go

```diff
@@ -16,13 +52,39 @@ const (
 //to find valid property names
 //
 func init() {
-	StatsExpvars.Add("changesFeeds_total", 0)
-	StatsExpvars.Add("changesFeeds_active", 0)
-	StatsExpvars.Add("requests_total", 0)
-	StatsExpvars.Add("requests_active", 0)
-	StatsExpvars.Add("revisionCache_hits", 0)
-	StatsExpvars.Add("revisionCache_misses", 0)
+	for _, stat := range globalStatNames {
+		StatsExpvars.Add(stat, 0)
+	}
```

Comment:
Also, several of the expvars you've got included in globalStatNames shouldn't be initialized as a simple int expvar in any case.

Reply:
Hmm, yeah, fair. I'm not sure of the best approach for this. We have some stats where we've set the precedent that they'll always be initialised as `0` - are we going to have to keep a list of those around?